README.md

```diff
@@ -201,3 +201,30 @@ This option is __disabled by default__. When this option is enabled, than for ea
 <pre><code>var reporter = new HtmlScreenshotReporter({
    preserveDirectory: true
 });</code></pre>
+
+### Multi-browser (optional)
+
+This option is __disabled by default__. When this option is enabled, than for each forked browser
+instance we will generate a screenshot and attach it to the report. Only thing you need to do to
+enable this option is save all browsers to the global `protractor.myInstances` object. For example:
+
+<pre><code>protractor.myInstances = {};
+
+describe('Testing multiple browsers', function() {
+
+  beforeAll(function() {
+    protractor.myInstances['firstBrowser'] = browser;
```

Comment:
I suggest that the default browser should be already in the map - that way we wouldn't need to have this as an option at all. It would just simple snap all the registered instances, with the default value set to `browser`.